lib/application-processing/resolvers.ts

```diff
@@ -874,15 +876,47 @@ export const updateApplicationProcessingGenerateInvoice: Resolver<
   }
 
   // Generate application invoice
-  generateApplicationInvoicePdf(
+  const pdfDoc = generateApplicationInvoicePdf(
     application,
     session,
     // TODO: Remove typecast when backend guard is implemented
     application.applicationProcessing.appNumber as number,
     invoice.invoiceNumber
   );
 
-  return { ok: true };
+  // Upload pdf to s3
+  const s3InvoiceKey = `invoices/${randomUUID()}/${invoice.invoiceNumber}.pdf`;
```

Comment:
Not sure what the file path should be. Right not its just the invoice number but it can be changes. Also added the random uuid for dev purposes so theres less collisions

Reply:
```suggestion
  const s3InvoiceKey = `rcd/invoices/${randomUUID()}/${invoice.invoiceNumber}.pdf`;
```
Let's put all application-related files in a base folder called `rcd/`. The rest of this file path looks good!